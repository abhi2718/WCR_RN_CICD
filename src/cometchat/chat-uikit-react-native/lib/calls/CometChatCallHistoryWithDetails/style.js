import { StyleSheet } from "react-native";
export const Style = StyleSheet.create({
    stackScreen: {
        position: "absolute",
        zIndex: 5,
        height: "100%",
        width: "100%"
    }
});
//# sourceMappingURL=style.js.map