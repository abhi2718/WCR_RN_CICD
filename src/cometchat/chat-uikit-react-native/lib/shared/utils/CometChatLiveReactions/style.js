import { StyleSheet } from "react-native";
export const Style = StyleSheet.create({
    reactionStyle: {
        marginRight: 10
    },
});
//# sourceMappingURL=style.js.map