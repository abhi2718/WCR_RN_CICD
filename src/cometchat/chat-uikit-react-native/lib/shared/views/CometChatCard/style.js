import { StyleSheet } from 'react-native';
export const Style = StyleSheet.create({
    container: {
        alignItems: "center",
        justifyContent: "space-between",
        padding: 16,
    },
    avatar: {
        height: "30%",
        width: "80%",
    }
});
//# sourceMappingURL=style.js.map