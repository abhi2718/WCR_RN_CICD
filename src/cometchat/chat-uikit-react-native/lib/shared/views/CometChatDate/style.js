//@ts-ignore
import { StyleSheet } from 'react-native';
export const Style = StyleSheet.create({
    textStyle: {
        textAlign: 'center',
    },
});
//# sourceMappingURL=style.js.map