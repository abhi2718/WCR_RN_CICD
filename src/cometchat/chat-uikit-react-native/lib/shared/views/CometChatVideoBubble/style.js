import { StyleSheet } from "react-native";
export const Style = StyleSheet.create({
    playIconPosition: {
        flex: 1,
        alignItems: "center",
        justifyContent: "center",
    }
});
//# sourceMappingURL=style.js.map