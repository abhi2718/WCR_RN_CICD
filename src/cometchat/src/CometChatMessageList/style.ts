import React from 'react';
import { StyleSheet } from 'react-native';
import { theme } from '../../../infrastructure/theme';
import { fonts } from '../../../infrastructure/theme/fonts';

export const Style = StyleSheet.create({
  nameStyle: {
    fontSize: 20,
    fontWeight: '500',
    color: 'rgba(20,20,20,0.9)',
  },
  newMessageIndicatorStyle: {
    position: 'absolute',
    zIndex: 20,
    padding: 4,
    paddingStart: 16,
    paddingEnd: 16,
    flexDirection: 'row',
    backgroundColor: 'rgb(51,155,255)',
    borderRadius: 16,
    alignSelf: 'center',
    marginTop: 16,
  },
  newMessageIndicatorImage: {
    tintColor: 'white',
    height: 16,
    width: 16,
    alignSelf: 'center',
    marginLeft: 8,
  },
  newMessageIndicatorText: {
    color: 'white',
    fontFamily: fonts.body,
  },
  msgContainerStyle: {
    flex: 1,
    overflow: 'hidden',
    width: '100%',
    justifyContent: 'center',
    alignItems: 'center',
  },
  msgTxtStyle: {
    justifyContent: 'center',
    alignItems: 'center',
    alignSelf: 'center',
    textAlign: 'center',
    width: '100%',
    color: 'rgba(20, 20, 20, 0.33)',
    fontFamily: fonts.body,
  },
  msgImgStyle: {
    tintColor: 'rgba(20, 20, 20, 0.58)',
  },
  stickyHeaderFooterStyle: {
    position: 'absolute',
    zIndex: 2,
    width: '100%',
  },
  groupName: {
    fontSize: theme.fontSizes.text,
    fontWeight: theme.fontWeights.bold,
    color: theme.colors.ui.text,
    textTransform: 'capitalize',
    fontFamily: fonts.body,
  },
});
