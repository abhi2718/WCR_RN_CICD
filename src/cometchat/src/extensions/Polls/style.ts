import React from 'react';
import { StyleSheet } from 'react-native';
import { fonts } from '../../../../infrastructure/theme/fonts';

export const styles = StyleSheet.create({
  container: { flex: 1, paddingHorizontal: 15 },
  textInput: {
    marginTop: 32,
    padding: 8,
    borderBottomWidth: 1,
    fontFamily: fonts.body,
  },
  textInputAnswers: {
    marginTop: 32,
    padding: 8,
    borderBottomWidth: 1,
    fontFamily: fonts.body,
  },
  errorContainer: {
    flexDirection: 'row',
    padding: 16,
    alignItems: 'center',
    marginTop: 25,
    borderRadius: 5,
  },
  errorImageContainer: {
    padding: 8,
    borderRadius: 100,
  },
  errorImage: {
    height: 24,
    width: 24,
    resizeMode: 'contain',
  },
  errorTextTitle: {
    marginStart: 8,
    fontFamily: fonts.body,
  },
  errorTextContainer: { flex: 1 },
  errorText: {
    marginStart: 8,
    fontFamily: fonts.body,
  },
  addAnswerButtonContainer: { marginTop: 30, marginBottom: 10 },
});
