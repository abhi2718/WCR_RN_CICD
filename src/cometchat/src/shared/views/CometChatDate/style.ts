//@ts-ignore
import { StyleSheet } from 'react-native';
import { fonts } from '../../../../../infrastructure/theme/fonts';

export const Style = StyleSheet.create({
  textStyle: {
    textAlign: 'auto',
    fontFamily: fonts.body,
  },
});
