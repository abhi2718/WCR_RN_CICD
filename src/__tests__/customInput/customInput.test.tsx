import React from 'react';
import { render,renderHook } from '@testing-library/react-native';
import { CustomInput } from '../../components/customInput';
import { useViewModal } from '../../components/customInput/useViewModal';
import { styles } from '../../components/customInput/index.style';

describe('Custom Input', () => {
    test('should render a custom input', () => {
        const tree = render(<CustomInput />);
        expect(tree.toJSON()).toMatchSnapshot()
    });
});
// 54 + 49 = 103
describe('Custom Input viewModal', () => {
    it('should match snapshot', () => {
    //   const { result } = renderHook(() => useViewModal());
    //     expect(result.current).toMatchSnapshot();
    const hookResult = useViewModal().toString();
        expect(hookResult).toMatchSnapshot();
    });
});

describe('Custom Input stylesheet', () => {
    it('should match snapshot', () => {
      expect(styles).toMatchSnapshot();
    });
});