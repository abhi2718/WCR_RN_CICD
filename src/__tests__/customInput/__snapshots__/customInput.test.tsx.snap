// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Input should render a custom input 1`] = `
<View>
  <Text>
    Hello world !
  </Text>
  <Text>
    Hello
  </Text>
</View>
`;

exports[`Custom Input stylesheet should match snapshot 1`] = `
{
  "container": {
    "backgroundColor": "green",
    "flex": 1,
  },
}
`;

exports[`Custom Input viewModal should match snapshot 1`] = `"[object Object]"`;
